pypdf/xmp.py

```diff
@@ -242,6 +242,32 @@ def __init__(self, stream: ContentStream) -> None:
         )[0]
         self.cache: dict[Any, Any] = {}
 
+    @classmethod
+    def create(cls) -> "XmpInformation":
+        """
+        Create a new XmpInformation object with minimal structure.
+
+        Returns:
+            A new XmpInformation instance with empty metadata fields.
+        """
+        minimal_xmp = f"""<?xpacket begin="\ufeff" id="W5M0MpCehiHzreSzNTczkc9d"?>
```

Comment:
Could you please make this an (internal) constant to reduce the complexity of the class?